### DISCUSSION TITLE
part of test results missing from feedback

### DISCUSSION THREAD

**Discussion by ukuvainik** [rubenarslan/formr.org#535] [Q&A] [answered]
Hi!

Nice new environment! I have a big5 personality test set up with feedback. People are complaining that conscientiousness scores are very low. Indeed, when I do a test run, the conscientiousness score is 0. The problem seems to be from the testing markdown file (attached) that conscientiousness items are not forwarded to the feedback script. However, if I look at the personality test .csv I download from the test, the responses are all there, even for the test person that the test script has picked up. 

here is the instrument
https://docs.google.com/spreadsheets/d/1VPeb20_8jOvehvwDn4B8Oj3yqN2mGc6avZBQA_4aPcg/edit#gid=0

How could i get the conscientiousness scores back?

Thanks!

Uku

[formr_rmarkdown.Rmd.txt](https://github.com/rubenarslan/formr.org/files/14704986/formr_rmarkdown.Rmd.txt)

**Comment by rubenarslan** [comment, marked as answer]
Hi Uku,
formr autodetects which variables you need by checking which surveys are mentioned in the source code and then which variables in the surveys are mentioned. It detects mentions of variable names (e.g., conscientiousness1) and variable name bases (e.g., conscientiousness). It doesn't detect `consc`, which is what you have. Just write it out, i.e. change this line


```
  mutate(Cscore=  rowSums(dplyr::select(.,starts_with("consc")),na.rm = T),
```

to

```
  mutate(Cscore=  rowSums(dplyr::select(.,starts_with("conscientiousness")),na.rm = T),
```
